### PR TITLE
create end points for post and delete answer

### PR DESCRIPTION
**Pull request by ccshah298**
# create endpoints post and delete for answers .. 

Please provide a short summary explaining what this PR is about.

Fixes # 103

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/routes/answers.router.js [hunk 1]
```diff
@@ -76,35 +76,70 @@ router.get('/:id', (req, res, next) => {
...
  *        description: Unexpected error.
  */
 router.post('/', (req, res) => {
   answersController
-    .createAnswer(req.body)
+    .createNewAnswer(req.body)
     .then((result) => res.json(result))
     .catch((error) => {
-      res.send(error.message);
+      console.log(error);
```
Comment (Enqira) [inline src/server/api/routes/answers.router.js:106]: Can you remove this log :)?

File: src/server/api/routes/answers.router.js [hunk 2]
```diff
@@ -76,35 +76,70 @@ router.get('/:id', (req, res, next) => {
...
+    .deleteAnswer(req.params.id, req)
+    .then((result) => {
+      if (result === 0) {
+        res.status(404).send('The answer ID you provided does not exist.');
+      } else {
+        res.json({ success: true });
+      }
+    })
+    .catch((error) => console.log(error));
+});
```
Comment (Enqira) [inline src/server/api/routes/answers.router.js:143]: Maybe you should send back the error message instead of loging it

**Review by Enqira — APPROVED** [review]
I tested it, works good, you just need to delete those 2 console logs

File: src/server/api/routes/answers.router.js (hunk 1 above)
Comment (ccshah298) [reply src/server/api/routes/answers.router.js:106]: Thanks for review @Enqira .. will change it :)

File: src/server/api/routes/answers.router.js [hunk 3]
```diff
@@ -64,10 +64,10 @@ router.get('/:id', (req, res, next) => {
  * /answers:
  *  post:
  *    tags:
- *    - answers
- *    summary: Create an answer
+ *     - answers
+ *    summary: Create a answer
```
Comment (dpfernandes) [inline src/server/api/routes/answers.router.js:68]: it'a "an Answer"

File: src/server/api/routes/answers.router.js [hunk 4]
```diff
@@ -76,35 +76,68 @@ router.get('/:id', (req, res, next) => {
...
  *      201:
- *        description: Answer created
+ *        description: answer created
  *      5XX:
  *        description: Unexpected error.
  */
 router.post('/', (req, res) => {
   answersController
-    .createAnswer(req.body)
+    .createNewAnswer(req.body)
```
Comment (dpfernandes) [inline src/server/api/routes/answers.router.js:103]: the word new is redundant, you don't create old or non-new stuff, so createAnswer is enough

File: src/server/api/routes/answers.router.js [hunk 5]
```diff
@@ -64,10 +64,10 @@ router.get('/:id', (req, res, next) => {
  * /answers:
  *  post:
  *    tags:
- *    - answers
- *    summary: Create an answer
+ *     - answers
+ *    summary: Create a answer
  *    description:
- *      Will create an answer.
+ *      Will create a answer.
```
Comment (dpfernandes) [inline src/server/api/routes/answers.router.js:70]: it'a "an Answer"

**Review by dpfernandes — DISMISSED** [review]
It looks good, It's mainly grammar documentation changes :)

**Review by TITHI1244 — APPROVED** [review]
Tried it, it works nicely. Good work @ccshah298

**Review by dpfernandes — APPROVED** [review]
👍

**Pull request merged**